drakrun/drakrun/config.py

```diff
@@ -0,0 +1,46 @@
+import os
+import json
+
+from dataclasses import dataclass
```

Comment:
We were using it already, right? I remember this is not present in some older Python versions and was causing problems. Since we support 3.7+ it should be fine though.

Reply:
Yes, I used it to build process tree.